timesync

```diff
@@ -99,7 +103,27 @@ def parse_all_args():
     return parser.parse_args()
 
 
+def read_config():
+    """Returns a dictionary representing the contents of the json file
+    $HOME/timesyncconf.json or a set of sensible defaults if that file does
+    not exist.
+    """
+    sensible_defaults = {
+        'url': 'http://localhost:3000',
+        'editor': 'vi',
+    }
+    # If $HOME isn't root base_path should be root.
+    base_path = os.environ.get('HOME') or '/'
+    path ='{}/.timesyncconf.json'.format(base_path)
+    if os.path.isfile(path):
+        with open(path) as f:
+            contents = json.load(f)
+            return sensible_defaults.update(contents)
```

Comment:
The update method does an in place update and returns nothing. Example below.
```python
>>> def test():
...  x = {'editor': 'vi'}
...  return x.update({'blah': 'foo'})
...
>>> test()
>>> print test()
None
>>>
```

To fix:
```python
sensible_defaults.update(contents)
return sensible_defaults
```